pkgs/desktops/gnome-3/extensions/appindicator/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "gnome-shell-extension-appindicator-${version}";
-  version = "22";
+  version = "23";
```

Comment:
Per [this issue](https://github.com/ubuntu/gnome-shell-extension-appindicator/issues/161), version 23 is broken on gnome 3.30, and support for 3.30 has been dropped. We shouldn't make this change until gnome is upgraded to 3.32.

Reply:
This would be great to include as part of #57027.

Reply:
Already done, but forgot about this pr.